sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blobs/tests/Samples/SamplesLiveTests.cs

```diff
@@ -48,7 +48,7 @@ public class SamplesLiveTests
         [TestCaseSource(nameof(SampleTestCases))]
         public async Task SmokeTestASample(IEventHubsBlobCheckpointSample sample)
         {
-            await using (EventHubScope eventHubScope = await EventHubScope.CreateAsync(2))
+            await using (EventHubScope eventHubScope = await EventHubScope.CreateIfMissingAsync(2))
```

Comment:
I'd prefer to keep the nomenclature as-is.   The scope is still being created, its a matter of whether the scope is based on a new Event Hubs instance or an existing one - which is an internal detail to  the scope, in my opinion.